pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -204,11 +212,16 @@ func isValidAggregationTemporality(metric pmetric.Metric) bool {
 
 func (c *prometheusConverter) addHistogramDataPoints(dataPoints pmetric.HistogramDataPointSlice,
 	resource pcommon.Resource, settings Settings, baseName string,
-) {
+) error {
+	var errs error
 	for x := 0; x < dataPoints.Len(); x++ {
 		pt := dataPoints.At(x)
 		timestamp := convertTimeStamp(pt.Timestamp())
-		baseLabels := createAttributes(resource, pt.Attributes(), settings.ExternalLabels, nil, false, c.labelNamer)
+		baseLabels, err := createAttributes(resource, pt.Attributes(), settings.ExternalLabels, nil, false, c.labelNamer)
+		if err != nil {
+			errs = multierr.Append(errs, err)
```

Comment:
I'm a little worried that we could generate a lot of errors as multierr has no limit/dedupe. I guess it's fine, we'll see if someone complains actually.

Reply:
Yeah, my biggest worry here is the allocation impact of increasing the error string, actually. The error should be super rare though, only if a translation results in empty string or just underscores

Reply:
why can't I resolve my own comment? anyway, I'm ok to resolve